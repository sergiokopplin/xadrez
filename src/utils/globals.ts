export const letterFromTo: { [key: string]: number } = {
  a: 1,
  b: 2,
  c: 3,
  d: 4,
  e: 5,
  f: 6,
  g: 7,
  h: 8
}

export const whitePieces = 'rnbqkp'

export const blackPieces = 'RNBQKP'

export const defaultRows: (string | null)[] = ['rnbqkbnr', 'pppppppp', null, null, null, null, 'PPPPPPPP', 'RNBQKBNR']
