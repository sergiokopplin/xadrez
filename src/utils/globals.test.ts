import { letterFromTo } from './globals'

describe('letterFromTo', () => {
  test('should return properly', () => {
    expect(letterFromTo).toMatchSnapshot()
  })
})
