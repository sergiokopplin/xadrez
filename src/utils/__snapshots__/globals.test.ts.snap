// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`letterFromTo should return properly 1`] = `
Object {
  "a": 1,
  "b": 2,
  "c": 3,
  "d": 4,
  "e": 5,
  "f": 6,
  "g": 7,
  "h": 8,
}
`;
